### PR TITLE
[WM-1657] Workflow order presentation

### PR DESCRIPTION
**Pull request by kpierre13**
…d date

### CODE REVIEW

**Review by mspector — APPROVED** [review]
this does seem simpler :)

File: service/src/main/java/bio/terra/cbas/dao/MethodDao.java [hunk 1]
```diff
@@ -34,7 +34,7 @@ public Method getMethod(UUID methodId) {
   }
 
   public List<Method> getMethods() {
-    String sql = "SELECT * FROM method";
+    String sql = "SELECT * FROM method GROUP BY method.method_id ORDER BY MIN(created) DESC";
```
Comment (cjllanwarne) [inline service/src/main/java/bio/terra/cbas/dao/MethodDao.java:37]: It feels really surprising that you need the group by here... are you sure this doesn't work?
```suggestion
    String sql = "SELECT * FROM method ORDER BY created DESC";
```

**Review by cjllanwarne — APPROVED** [review]
LGTM albeit with an eyebrow raised...

File: service/src/main/java/bio/terra/cbas/dao/MethodDao.java (hunk 1 above)
Comment (kpierre13) [reply service/src/main/java/bio/terra/cbas/dao/MethodDao.java]: This totally works. I don't know why I thought I needed to use `MIN()`.  That was forcing me to use the `GROUP BY` method.

**Review by cjllanwarne — APPROVED** [review]
Re-thumb from me!

**Pull request merged**